src/storage/client/StorageClient.inl

```diff
@@ -158,6 +157,49 @@ folly::SemiFuture<StorageRpcResponse<Response>> StorageClient::collectResponse(
     return context->promise.getSemiFuture();
 }
 
+
+template<class Request, class RemoteFunc, class Response>
+folly::Future<StatusOr<Response>> StorageClient::getResponse(
```

Comment:
I noticed the function has been supported in #868

Reply:
He copy mine, I will rebase master. :)